pkg/build/api/types.go

```diff
@@ -86,6 +95,10 @@ type BuildStatus struct {
 	// Cancelled describes if a cancel event was triggered for the build.
 	Cancelled bool
 
+	// Skipped indicates that this build was marked as skipped. The build will be
```

Comment:
@mfojtik We also discussed in the card(I think?) having a label on builds that are queued so you could do a delete by label to delete all queued builds...